src/btree/bt_delete.c

```diff
@@ -146,11 +147,11 @@ __wt_delete_page(WT_SESSION_IMPL *session, WT_REF *ref, bool *skipp)
 }
 
 /*
- * __wt_delete_page_rollback --
- *     Abort fast-truncate operations.
+ * __wt_delete_page_abort --
+ *     Transaction abort for a fast-truncate operation.
```

Comment:
WT terminology is "rollback", not "abort".

Reply:
Fair enough, will revert. Also, turns out the chief user of the word "abort" (other than the function to crash) is RTS, so it doesn't help anyway... fixed in [2e5af24](https://github.com/wiredtiger/wiredtiger/pull/7909/commits/2e5af24e34f8557ee0cb39ab8a1f28f054c35c84)